configure.ac

```diff
@@ -1,6 +1,6 @@
 #  Copyright (C) 2020 Tobias Bossert
 
-AC_INIT([wg-meta],m4_esyscmd([tr -d '\n' < VERSION]),[tobias.bossert@fastpath.ch])
```

Comment:
do you still need configure? if so, I would try to NOT keep the version number in 2 places.

Reply:
Removed automake altogether, I think `ExtUtils::MakeMaker` is sufficient here. The version number is now just in the `VERSION` file an propagates to the occurrences in the code when calling `make`.